index.js

```diff
@@ -1111,7 +1112,7 @@ class GithubScm extends Scm {
             const pullRequestComment = await this.breaker.runCommand({
                 action: 'createComment',
                 scopeType: 'issues',
-                token,
+                token: this.config.token, // need to use a token with public_repo permissions
```

Comment:
I feel token is too generic, can name be made specific for it's purpose ? commentUserToken or something ?

Reply:
updated